docs/docs/target/kubernetes.md

```diff
@@ -26,6 +26,54 @@ Kubernetes resource definition is scanned for:
 - Misconfigurations
 - Exposed secrets
 
+## Required roles
+To successfully scan a Kubernetes cluster, `trivy kubernetes` subcommand must be executed under a role that has some specific permissions.
+
+Without `node collecor` (with flag `--disable-node-collector`) the role must have `list` verb for all resources ("*") inside next API groups: core (""), "apps", "batch","networking.k8s.io", "rbac.authorization.k8s.io":
+```yaml
+- apiGroups: [""]
+  resources: ["*"]
+  verbs: ["list"]
+- apiGroups: ["apps", "batch", "networking.k8s.io", "rbac.authorization.k8s.io"]
+  resources: ["*"]
+  verbs: ["list"]
+```
+
+If `node collector` is enabled (by default), the cluster role must have more permissions than:
```

Comment:
Do you mean to say this is the minimum set of permissions required to run `node collector`?

Reply:
no, the list for `node-collector` has more permissions